app/models/book.rb

```diff
@@ -27,15 +27,7 @@ def manifests
   end
 
   def range_labels_from_manifests
```

Comment:
can you add a comment as to how the ocr text is related to the range labels from manifests?

Reply:
or change the method name?

Reply:
I think the job needs to be renamed to IndexRangeLabelsJob or something, because it's indexing range labels, not OCR.

Reply:
It's -going- to be OCR at one point, and range labels are kinda like that.

Reply:
OK, that makes sense. Can you add a comment or something to clarify the disconnect, so it's clear what's happening now vs. what we plan on doing?